aslam_cv_matcher/include/aslam/matcher/gyro-two-frame-matcher.h

```diff
@@ -185,17 +185,17 @@ class GyroTwoFrameMatcher {
   // Two descriptors could match if they pass the Lowe ratio test.
   static constexpr float kLoweRatio = 0.8f;
   // Small image space distances for keypoint matches.
-  static constexpr int kSmallSearchDistance = 10;
+  const int kSmallSearchDistance;
```

Comment:
wrong variable name style (here and below)
note: this is (no longer) a compile time constant

Reply:
and might make sense to add units (px)